app/controllers/atmosphere/api/v1/virtual_machine_flavors_controller.rb

```diff
@@ -20,6 +20,7 @@ def index
 
           if optimizer_query?
             tmpls = VirtualMachineTemplate.active.on_active_tenant
+              .includes(tenants: [virtual_machine_flavors: [:os_families, :flavor_os_families]])
```

Comment:
Line is too long. [96/80]<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.